cms/static/js/spec/models/group_configuration_spec.js

```diff
@@ -48,10 +48,12 @@ define([
             });
 
             it('should be able to reset itself', function() {
-                this.model.set('name', 'foobar');
-                this.model.reset();
+                var originalName = 'Original Name',
+                    model = new GroupConfigurationModel({name: originalName});
```

Comment:
Note to reviewers: not using `this.model`, since it has a blank name, which does not validate on `reset()`.